docs/readmes/orc8r/development/testing_tips.md

```diff
@@ -16,7 +16,9 @@ subproject implements, e.g. `orc8r/cloud/go/Makefile`.
 
 The set of subprojects is determined by the `MAGMA_MODULES` environment
 variable, which is defined during the container image build process. The
-default subprojects include `orc8r`, `lte`, `feg`, etc.
+default subprojects include `orc8r`, `lte`, `feg`, etc. You can specify which
```

Comment:
Move your changes to a separate paragraph

"By default, all subprojects are included in the build. If desired, you can use e.g. `build.py --deployment orc8r` to build a deployment-specific set of subprojects. This can reduce build time by up to 50%."

Reply:
Got it, I'll keep that in mind